### ISSUE TITLE
Check the re-OCR quality of motions

### ISSUE THREAD

**Issue by fredrik1984** [welfare-state-analytics/riksdagen-corpus#393]
Liam has re-OCRed all motions from 1867–1970. We need to check the OCR quality of this work.

**Comment by MansMeg** [comment]
Yes. I think we should check it when we have moved over the text to the new motion format.

Måns
